Sprint-3/implement/rotate-char.js

```diff
@@ -41,3 +41,50 @@ console.log(rotateCharacter("7", 5)); // Output: "7" (unchanged, not a letter)
 // And the function should return the rotated character as a string (e.g., 'z' rotated by 3 should become 'c', 'Z' rotated by 3 should become 'C').
 console.log(rotateCharacter("z", 1)); // Output: "a" (preserves case, but wraps around)
 console.log(rotateCharacter("Y", 2)); // Output: "A" (preserves case, but wraps around)
+_______________________________________SOLUTION___________________________________________
+
+function rotateCharacter(char, shift) {
+    // Check if the character is a lowercase letter
+    if (char >= 'a' && char <= 'z') {
+        const base = 'a'.charCodeAt(0);
+        // Calculate rotated character within lowercase range
+        return String.fromCharCode(base + (char.charCodeAt(0) - base + shift) % 26);
```

Comment:
Extra Question: 
If `shift` is allowed to be a negative number (says -100) to represent a rotation in the opposite direction. How would you change your code to support such feature?